src/chrome/content/rules/Gumtree.com.xml

```diff
@@ -0,0 +1,546 @@
+		- plastererwww	(m)
+		- playedwww	(m)
+		- playwww	(m)
+		- plymouth	(m)
+		- plzwww	(m)
+		- pokerwww	(m)
+		- policewww	(m)
+		- poohwww	(m)
+		- popularwww	(m)
+		- portawww	(m)
+		- portsmouth	(m)
+		- postwww	(m)
+		- poundswww	(m)
+		- pre	(m)
+		- prelovedcars	(m)
+		- premiumwww	(m)
+		- preparewww	(m)
+		- prerelease	(m)
+		- preth	(m)
+		- priceswww	(m)
+		- princesswww	(m)
+		- probablywww	(m)
+		- problem	(m)
+		- protonwww	(m)
+		- providerswww	(m)
+		- my.qa6	(m)
+		- quality-used	(m)
+		- quietwww	(m)
+		- rattanwww	(m)
+		- www.reading	(m)
+		- recyclerwww	(m)
+		- refinewww	(m)
+		- regulatorwww	(m)
+		- release	(m)
+		- renownedwww	(m)
+		- rentsmart	(m)
+		- rentwww	(m)
+		- repairwww	(m)
+		- replacewww	(m)
+		- resultswww	(m)
+		- rightwww	(m)
+		- rio-motors	(m)
+		- salesforce-gateway	(m)
+		- salonwww	(m)
+		- sansuiwww	(m)
+		- santawww	(m)
+		- sawwww	(m)
+		- schucowww	(m)
+		- scooter	(m)
+		- www.scotland	(m)
+		- sdwww	(m)
+		- sdynet	(m)
+		- search	(m)
+		- secure	(m)
+		- seeminglywww	(m)
+		- sellswww	(m)
+		- semiwww	(m)
+		- server	(m)
+		- servicewww	(m)
+		- integration.sf	(r)
+		- sfgw.sf	(r)
+		- sg	(m)
+		- shedwww	(m)
+		- sheffield	(m)
+		- shiatsuwww	(m)
+		- shirtswww	(m)
+		- showerwww	(m)
+		- showww	(m)
+		- sidewayswww	(m)
+		- sidney	(m)
+		- www.sidney	(m)
+		- sillywww	(m)
+		- singerwww	(m)
+		- site	(m)
+		- sites	(m)
+		- sitewww	(m)
+		- sleepswww	(m)
+		- slingswww	(m)
+		- smallerwww	(m)
+		- snarewww	(m)
+		- solewww	(m)
+		- soonnhttpwww	(m)
+		- southampton	(m)
+		- southeastwww	(m)
+		- southend	(m)
+		- southwww	(m)
+		- spacesuitwww	(m)
+		- speakwww	(m)
+		- ssl	(m)
+		- startwww	(m)
+		- stats	(m)
+		- stoke	(m)
+		- store	(m)
+		- storeswww	(m)
+		- storywww	(m)
+		- stringwww	(m)
+		- stuth	(m)
+		- subdomain	(m)
+		- suitwww	(m)
+		- sunderland	(m)
+		- sunwww	(m)
+		- survey	(m)
+		- suzukiwww	(m)
+		- svn	(m)
+		- swansea	(m)
+		- swapup	(m)
+		- swapwww	(m)
+		- swidon	(m)
+		- swindon	(m)
+		- swindoon	(m)
+		- sydne	(m)
+		- tasamnia	(m)
+		- tecwww	(m)
+		- teethingwww	(m)
+		- test	(m)
+		- testwww	(m)
+		- thatwww	(m)
+		- thoughtwww	(m)
+		- thrill-www	(m)
+		- throws	(m)
+		- tipperland	(m)
+		- to	(m)
+		- tonicwww	(m)
+		- toplymouth	(m)
+		- toppingswww	(m)
+		- toronto	(m)
+		- townsville	(m)
+		- trackswww	(m)
+		- tryingwww	(m)
+		- tweedheads	(m)
+		- tyreswww	(m)
+		- usbwww	(m)
+		- uswww	(m)
+		- www.uwc	(m)
+		- uwww	(m)
+		- valkyriewww	(m)
+		- vampirewww	(m)
+		- vancouver	(m)
+		- vic	(m)
+		- videos	(m)
+		- australia.wa	(m)
+		- wb-www	(m)
+		- web-ww	(m)
+		- webmail	(m)
+		- www.webmail	(m)
+		- windwww	(m)
+		- wollongong	(m)
+		- writtenwww	(m)
+		- wsss	(m)
+		- wvw	(m)
+		- wvww	(m)
+		- ww	(m)
+		- ww2	(m)
+		- wwew	(m)
+		- wwip3	(m)
+		- www2	(m)
+		- www3	(m)
+		- www4	(m)
+		- wwww	(m)
+		- www.wwwwww	(m)
+		- xmaswww	(m)
+		- xn--www-rp0a	(m)
+		- xr16www	(m)
+		- yearswww	(m)
+		- york	(m)
+		- yzfwww	(m)
+		- ztrwww	(m)
+		- zxwww	(m)
```

Comment:
There is a wildcard DNS record, so it makes no sense to list random subdomains here.